group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -1819,6 +1822,10 @@ public void onLoaded() {
                             ". Rebalancing in order to give a chance for consumers to commit offsets");
                     }
                     break;
+
+                default:
+                    log.warn("Unexpected group type detected while loading: " + group.type());
```

Comment:
Could we say: `Loaded group {} with an unknown type {}.`?

Reply:
Does it still load the group if the type is unknown though?

Reply:
The group is in `groups` so it is already loaded at this point.

Reply:
Oh yeah my bad, morning brain :(